android/src/main/java/com/twiliorn/library/niceday/NDHelper.java

```diff
@@ -0,0 +1,47 @@
+package com.twiliorn.library.niceday;
+
+import android.support.annotation.Nullable;
+import android.util.Log;
+
+import com.twilio.video.TrackPriority;
+import com.twilio.video.VideoDimensions;
+
+public class NDHelper {
```

Comment:
Similar with this one NDHelper. Explain or detail why is this needed in our end to help understand

Reply:
@jagamypriera Can you respond to this?